src/App.js

```diff
@@ -1,36 +1,44 @@
 import React, { Component,lazy,Suspense } from 'react';
-import {withRouter,Route,Switch} from 'react-router-dom'
+import {withRouter,Route} from 'react-router-dom'
 import Blog from './containers/Blog/Blog';
 import {connect} from 'react-redux';
+import * as actions from './store/actions/index';
 const Auth = lazy(() => import('./containers/Auth/Auth'));
 const Logout = lazy(()=>import('./containers/Auth/Logout/Logout'));
```

Comment:
import this lazy in the map not separately by pasing `path` as paramerte.

Reply:
Sir,I have added lazy in the map